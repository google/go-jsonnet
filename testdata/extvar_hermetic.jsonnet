local x = 42; std.extVar("UndeclaredX")
