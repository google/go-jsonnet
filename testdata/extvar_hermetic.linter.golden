../testdata/extvar_hermetic:1:7-13 Unused variable: x

local x = 42; std.extVar("UndeclaredX")


